### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by MnM001** [sameersbn/docker-invoiceplane#27]
Hi,

I have installed this docker (thank you for your efforts so far) and it works fine.
However I wonder if some improvements can be done to it?

I have a mariadb docker running already and ideally I will like the InvoicePlane DB to be on that. I have tried to make it work but there are some issues. It is to what user ID and group ID my existing docker stack uses and with this docker just ignoring them if defined.

All the other dockers in my stack are created/started with a particular PUID and PGID - so permissions on the volumes are specific to that.
The Invoiceplane docker ignores PUID and PGID and the created containers and files are owned by a mixture of www-data and systemd-resolve user. For group I can see systemd-journal.

Because of this the rest of docker stack cannot see the Invoiceplane area and vice-versa.

It will be good to allow PUID and PGID to allow integration of this docker with existing stacks and not to have to run separate SQL, NGINX dockers.

Thanks

**Comment by geek-baba** [comment]
Yes setting uid and gid will let me deploy this in my kubernetes cluster with trafik ingress and mariadb endpoint.
